aws/elbv2_test.go

```diff
@@ -8,11 +8,14 @@ import (
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/aws/aws-sdk-go/service/elbv2"
+	"github.com/gruntwork-io/cloud-nuke/config"
+	"github.com/gruntwork-io/cloud-nuke/logging"
 	"github.com/gruntwork-io/cloud-nuke/util"
 	"github.com/gruntwork-io/go-commons/collections"
 	"github.com/gruntwork-io/go-commons/errors"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"regexp"
```

Comment:
In this PR and in other noticed that import order is different how `goimports` likes, maybe make sense to run on changed files `goimports -w ` to have files the code base in the same format

Reply:
Good call - I've corrected the imports across all PRs. Thanks!